### DISCUSSION TITLE
No github release  for gix-transport 0.42

### DISCUSSION THREAD

**Discussion by NobodyXu** [GitoxideLabs/gitoxide#1343] [General]
There's usually a git tag and github release for every new release of crates.

However, I didn't find the github release for gix-transport 0.42

**Comment by Byron** [comment]
Sorry for that - there was human error that prevented the GitHub release creation, so the releases that are there have been created manually. And now, there also is [a `gix-transport` release page](https://github.com/Byron/gitoxide/releases/tag/gix-transport-v0.42.0), to conclude creating GH releases for the most important crates of this (early) `gix` release.
